### PR TITLE
Vault style fixes

### PR DESCRIPTION
**Pull request by Seroxdesign**
Fixes #

**Changes proposed in this pull request:**

-Positions of buttons fixed
-Block buttons for users that don't give allowance
-Block Negative numbers from being place into input field  (user can put letters but the pay button won't work for them)
-Changed casing on buttons

-applied changes to mobile

### CODE REVIEW

File: src/components/banks/VaultTransaction/index.tsx [hunk 1]
```diff
@@ -112,7 +113,9 @@ export const VaultTransaction: FC<Props> = ({
                   <TextInput
                     type="number"
                     value={value}
+                    min={0}
```
Comment (bigint) [inline src/components/banks/VaultTransaction/index.tsx:116]: Have you tried with negative numbers by typing it or scrolling in the mouse?

File: src/components/banks/VaultTransaction/index.tsx [hunk 2]
```diff
@@ -112,7 +113,9 @@ export const VaultTransaction: FC<Props> = ({
...
                     onChange={onChange}
+                    onKeyDown={blockInvalidChar}
```
Comment (bigint) [inline src/components/banks/VaultTransaction/index.tsx:118]: kudos for adding this 👏🏻

File: src/components/banks/VaultTransaction/index.tsx (hunk 1 above)
Comment (Seroxdesign) [reply src/components/banks/VaultTransaction/index.tsx:116]: I've tried both

File: src/components/banks/VaultTransaction/index.tsx (hunk 2 above)
Comment (Seroxdesign) [reply src/components/banks/VaultTransaction/index.tsx:118]: This here stops users from typing in negative numbers or e.

**Comment by josealonso** [comment]
I just reviewed and the only (minor) issue I find is that the casing is only fixed inside the repay section. The remaining proposed changes are fixed.

**Comment by josealonso** [comment]
I see the same casing on the buttons, but it's changed on the code, it's must be my cache.
But the text must be prepared for using any language, not only English.

**Comment by josealonso** [comment]
I approve this pull request. I'm sorry I don't know how to do it properly.
Translating all the labels in the app is important, even though some labels are not prepared for translation. That's a different issue.

**Comment by Seroxdesign** [comment]
I agree, I'll have that added as an issue.  Glad I can do this one.

**Review by josealonso — APPROVED** [review]
I checked all the proposed changes and corrections and this can be merged.

**Pull request merged**